docs/span_metadata.md

```diff
@@ -228,13 +228,16 @@ Type | `grpc`
 Name | Required |
 ---------|----------------|
 component | `Grpc`
-grpc.method.kind | Yes
-grpc.method.name | Yes
-grpc.method.package | Yes
-grpc.method.path | Yes
-grpc.method.service | Yes
-grpc.status.code | Yes
+rpc.grpc.kind | Yes
+rpc.grpc.package | Yes
+rpc.grpc.path | Yes
+rpc.method | Yes
+rpc.service | Yes
```

Comment:
While we're renaming everything, we should also add the required `rpc.system=grpc` tag to these spans

Reply:
I suppose we can add that separately if you want to focus this PR on the renames

Reply:
done in 5af1578cf2109ef962326c9835b19c996e47cba4